dietpi-install.sh

```diff
@@ -14,13 +14,65 @@ cleanup() {
 # Trap Ctrl+C and other interrupts
 trap cleanup INT TERM
 
-# Variables
-IMAGE_URL=$(whiptail --inputbox 'Enter the URL for the DietPi image (default: https://dietpi.com/downloads/images/DietPi_Proxmox-x86_64-Trixie.qcow2.xz):' 8 78 'https://dietpi.com/downloads/images/DietPi_Proxmox-x86_64-Trixie.qcow2.xz' --title 'DietPi Installation' 3>&1 1>&2 2>&3)
-
-# Check if user cancelled
-if [ $? -ne 0 ]; then
-    cleanup
-fi
+# Select DietPi OS Version
+while true; do
+    OS_VERSION=$(whiptail --title "DietPi Installation" --menu "Select DietPi image:" 22 65 13 \
+        "."               "───────── Debian 13 Trixie ─────────" \
+        "trixie"          "Standard (Recommended)" \
+        "trixie-uefi"     "UEFI Boot" \
+        ".."              "───────── Debian 12 Bookworm ───────" \
+        "bookworm"        "Standard" \
+        "bookworm-uefi"   "UEFI Boot" \
+        "..."             "───────── Debian 14 Forky ──────────" \
+        "forky"           "Standard (Testing)" \
+        "forky-uefi"      "UEFI Boot (Testing)" \
+        "...."            "─────────────────────────────────────" \
+        "custom"          "Custom URL" 3>&1 1>&2 2>&3)
+
+    # Check if user cancelled
+    if [ $? -ne 0 ]; then
+        cleanup
+    fi
+
+    # If separator selected, show menu again
+    case "$OS_VERSION" in
+        .|..|...|....) continue ;;
+        *) break ;;
+    esac
```

Comment:
Prefer single quotes for strings without variables. And no need to use any value for separator lines, which makes the check a bit simpler.
```suggestion
    OS_VERSION=$(whiptail --title 'DietPi Installation' --menu 'Select DietPi image:' 19 65 11 \
        ''                '───────── Debian 13 Trixie ─────────' \
        'trixie'          'Standard (Recommended)' \
        'trixie-uefi'     'UEFI Boot' \
        ''                '───────── Debian 12 Bookworm ───────' \
        'bookworm'        'Standard' \
        'bookworm-uefi'   'UEFI Boot' \
        ''                '───────── Debian 14 Forky ──────────' \
        'forky'           'Standard (Testing)' \
        'forky-uefi'      'UEFI Boot (Testing)' \
        ''                '────────────────────────────────────' \
        'custom'          'Custom URL' 3>&1 1>&2 2>&3)

    # Check if user cancelled
    if [ $? -ne 0 ]; then
        cleanup
    fi

    # If separator selected, show menu again
    if [ -n "$OS_VERSION" ]; then
        break
    fi
```
This also narrows down the menu height to what is actually needed, and aligns the length of the last separator line.